src/index.njk

```diff
@@ -5,10 +5,14 @@ title: SB Hacks VII
 {% extends "layouts/base.njk" %}
 
 {% block body %}
-<main class="h-screen bg-gradient-to-b lg:p-10 from-sea-light to-sea-dark rendering-pixelated overflow-x-hidden">
-  <div class="h-full flex items-center justify-center flex-col lg:flex-row">
+<main class="h-screen rendering-pixelated overflow-x-hidden">
+  <div class="lg:p-10 bg-gradient-to-b from-sea-light to-sea-dark h-full flex items-center justify-center flex-col lg:flex-row">
```

Comment:
Just curious, what is the extra padding here for?

Reply:
This padding was originally on the main div, since i moved the gradient tags from the main to their respective divs the padding in the main div was messing with the gradient. I moved the padding there so that the art was consistent